src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
 // write your code here
+const bodyElement = document.body;
+
+const population = [...bodyElement.querySelectorAll('.population')];
+const textTotal = bodyElement.querySelector('.total-population');
+const textAverage = bodyElement.querySelector('.average-population');
+
+const value = population.map((elem) => +elem.textContent.replaceAll(',', ''));
```

Comment:
The use of `replaceAll` might not be supported in all environments. Consider using `replace` with a global regular expression instead for better compatibility.